src/Map/src/Exception/UnableToNormalizeOptionsException.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Exception;
+
+use Symfony\UX\Map\MapOptionsInterface;
+
+final class UnableToNormalizeOptionsException extends LogicException
+{
+    public function __construct(string $message)
+    {
+        parent::__construct(\sprintf('Unable to normalize the map options: %s', $message));
+    }
+
+    /**
+     * @param class-string<MapOptionsInterface> $classOptions
```

Comment:
I'm having a bit of a freeze every time i read this class 😅 

What do you think we rename it optionsClass everywhere ?

Reply:
Sure!